Detectors/TPC/qc/src/Clusters.cxx

```diff
@@ -38,11 +38,27 @@ bool Clusters::processCluster(const T& cluster, const o2::tpc::Sector sector, co
     LOGP(warning, "calling denormalize() before filling");
   }
 
+  const auto& mapper = Mapper::instance();
+
   const int nROC = row < 63 ? int(sector) : int(sector) + 36;
   const int rocRow = row < 63 ? row : row - 63;
 
   const float pad = cluster.getPad();
 
+  size_t position = mapper.getPadNumber(mNClusters.getCalArray(nROC).getPadSubset(), mNClusters.getCalArray(nROC).getPadSubsetNumber(), rocRow, pad);
+
+  if ((nROC < mapper.getNumberOfIROCs())) { // IROC
+    if (position >= mapper.getPadsInIROC()) {
+      LOG(error) << "IROC out of range: pad position " << position << "\tROC " << nROC << "\tIROC has " << mapper.getPadsInIROC() << " pads";
+      return false;
+    }
+  } else if ((nROC >= mapper.getNumberOfIROCs())) { // OROC
```

Comment:
The number is the same, but should this not be `getNumberOfOROCs`? See https://github.com/AliceO2Group/AliceO2/blob/3e46ea52b652e4c706b6dda8398fc355c536dc67/Detectors/TPC/base/include/TPCBase/Mapper.h#L399-L400

Reply:
the nROC here is defined so that from zero to 36 it is IROCs and 36 to 72 it designates OROCs, so here the 
`(nROC >= mapper.getNumberOfIROCs()` 
is accurate. If wanted, to make it more clear I can change it to 
`(nROC >= mapper.getNumberOfIROCs() && nROC < mapper.getNumberOfIROCs()+mapper.getNumberOfOROCs()`
but shouldn't be necessary for the logic.

Reply:
Ah true, I overlooked that it is just an `else` to the condition on line 50. But then, isn't just `else {` enough? without the check `if ((nROC >= mapper.getNumberOfIROCs()))`?

Reply:
yes you are right, `else if` instead of `else` is redundant here

Reply:
Hello @bulukutlu , @wiechula , 
It woudl be good to have this PR finalized. If you agree to remove the `else`, can you please do so and then we can habve the PR merged? In this way we can have back the TPC cluster QC in production. 
Thanks!